extensions/roulette/database/redis_db.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+from Redis.redis_config import get_redis
+from ..config import config
+from .action import fetch_timeout_roles
+
+from datetime import datetime, timedelta
+from discord import Member, Message
+from typing import Set
+
+_redis_client = get_redis()
+
+logger = logging.getLogger("roulette.database")
+
+
+# Mute functions
+# TODO: Check for successful update
+async def _upload_timeout(duration: timedelta,
+                          message: Message,
+                          target: Member) -> int:
```

Comment:
nitpicky, but could we add an Enum for return values, like SUCCESS and FAILURE? imo there's no need to expose the internal workings of Redis to client calling code (and make them have to understand what Redis respond codes are)